.github/workflows/build.yaml

```diff
@@ -199,6 +199,8 @@ jobs:
     # - https://github.com/ScaCap/action-surefire-report/issues/17
     env:
       NODE_OPTIONS: '--max_old_space_size=4096'
+      # Total Number of Slices
+      SLICE_TOTAL_SLICES: 11
```

Comment:
Please add a note to indicate the value needs to match the total number if items listed in the `strategy.matrix.slice` element.